packages/form-js-viewer/src/render/components/form-fields/parts/DropdownList.js

```diff
@@ -0,0 +1,100 @@
+import { useState, useEffect, useCallback, useRef, useMemo } from 'preact/hooks';
```

Comment:
Same as for the hook: as it should be reusable, should we add unit tests for this one?